docs/platform/porting/system.md

```diff
@@ -2540,3 +2540,353 @@ Example:
+
+-  **fd_reporting_period**  
+     Minimum period of time in seconds between reporting the subsequent abnormalities for the same process.
+
+     Default value: `-1`
+
+The following is the example configuration:
+```
+{
+    "global":{
+        "monitor": 1,
+        "print_current": 0,
+        "kill": 0,
+        "report": 0,
+        "sampling_rate": 1,
+
+        "cpu_limit_avg": 4,
+        "cpu_avg_period": 10,
+        "cpu_limit_peak": 4,
+        "cpu_reporting_period": 20,
+
+        "mem_limit_avg": 1,
+        "mem_avg_period": 10,
+        "mem_limit_peak": 1,
+        "mem_reporting_period": 20,
+
+        "io_limit_avg": 100.0,
+        "io_avg_period": 10,
+        "io_limit_peak": 100.0,
+        "io_reporting_period": 20,
+
+        "fd_limit": 1000,
+        "fd_reporting_period": -1,
+    },
+
+    "foreground_native_app":{
+        "cpu_limit_avg": 0.6,
+    },
+
+    "background_native_app":{
+        "cpu_limit_avg": 0.5,
+    },
+
+    "other_service":{
+        "cpu_limit_avg": 0.7,
+    },
+
+    "test-stability":{
+        "monitor": 1,
+        "cpu_limit_avg": 0.7,
+        "cpu_limit_peak": 0.9,
+        "mem_limit_peak": 0.3,
+    },
+
+    "sleep":{
+        "monitor": 0,
+    },
+
+    "stability-monitor":{
+        "monitor": 0,
+        "print_current": 0,
+        "cpu_limit_avg": 0.02,
+        "cpu_avg_period": 10,
+    },
+
+    "crash-manager":{
+        "monitor": 0,
+        "apply_to_children": 1,
+    },
+
+    "9zWvGSYU8Z.calendar": {
+        "print_current": 1,
+    }
+}
+```
+
+#### D-Bus signal
+
+When the abnormality is detected, the stability monitor sends one of the following D-Bus signals, depending on the module or parameter that detected or caused the abnormality:
+
+-  **cpu**
+
+    Signature: `isdda{sv}`  
+    Member: `AbnormalityDetected`  
+    Interface: `org.tizen.abnormality.cpu.relative`  
+    Path: `/Org/Tizen/StabilityMonitor/tsm_cpu`  
+    Data structure:  
+
+    Data                  | Type   | Description
+    ----------------------|--------|------------
+    pid                   | INT32  | Process ID
+    limitType             | STRING | Either `peak` or `avg`
+    cpuRelative           | DOUBLE | Actual value that caused the abnormality
+    maxAllowedCpuRelative | DOUBLE | Allowed value
+    array                 | ARRAY  | Additional parameters
+
+-  **mem**
+
+    Signature: `isdda{sv}`  
+    Member: `AbnormalityDetected`  
+    Interface: `org.tizen.abnormality.mem.relative`  
+    Path: `/Org/Tizen/StabilityMonitor/tsm_mem`  
+    Data structure:  
+
+    Data                  | Type   | Description
+    ----------------------|--------|------------
+    pid                   | INT32  | Process ID
+    limitType             | STRING | Either `peak` or `avg`
+    memRelative           | DOUBLE | Actual value that caused the abnormality
+    maxAllowedMemRelative | DOUBLE | Allowed value
+    array                 | ARRAY  | Additional parameters
+
+-  **io**
```

Comment:
We usually write I/O, I think this is a rule-set name so we can use io. 

Am I right?  please clear my doubt.

Reply:
This is just description, I've changed it.